scripts/run.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# Default environment file
+env_file=".env-dev"
+JB_ENGINE_VERSION=0.0.1
+
+# Process command-line arguments
+while getopts 'e:v:-:' flag; do
+  echo "Inside Jugalbandi Manager Configuration for WSL2"
+  case "${flag}" in
+    e) env_file="${OPTARG}" ;;
+    v) JB_ENGINE_VERSION="${OPTARG}" ;;
+    -) case "${OPTARG}" in
+            stage)
+                stage=true
+                ;;
+            *)
+                echo "Unknown option --${OPTARG}" >&2
+                exit 1
+                ;;
+        esac
+        ;;
+    *) echo "Usage: $0 [-e env_file] [-v JB_ENGINE_VERSION] [service1 service2 ...]"
+       exit 1 ;;
+  esac
+done
+
+# Remove processed options from the arguments list
+shift $((OPTIND -1))
+
+# Additional build steps if needed
+cd ../Jugalbandi-Studio-Engine/ && ./build.sh "${JB_ENGINE_VERSION}"
+cd -
+
+cd ../PwR-NL2DSL/ && poetry install && poetry build
+cd -
+cp ../PwR-NL2DSL/dist/*.whl ./server/dist  
+
+cd ../PwR-Studio/lib && poetry install && poetry build
+cd -
+cp ../PwR-Studio/lib/dist/*.whl ./server/dist  
+
+
+# Create necessary directories
+# mkdir -p ./media
+mkdir -p ./server/dist
+
+# Determine host settings based on environment (WSL or Docker)
+if grep -qi microsoft /proc/version && grep -q WSL2 /proc/version; then
+    echo "Inside PwR Studio Configuration for WSL2"
+    JBHOST=$(hostname -I | awk '{print $1}')
```

Comment:
Clean up these. Use only JBHOST

Remove port number from JBHOST and set it correctly in .env file